core/spec/models/spree/wallet/add_payment_sources_to_wallet_spec.rb

```diff
@@ -13,5 +15,18 @@
         order.user.wallet.wallet_payment_sources.count
       }.by(1)
     end
+
+    it 'keeps the payment source as default when used' do
+      user = order.user
+      user.wallet.add(create(:credit_card, user: user))
+      user.wallet.add(create(:credit_card, user: user))
+      user.wallet.default_wallet_payment_source = user.wallet_payment_sources.first
+      order.payments.first.update!(source: user.wallet.default_wallet_payment_source.payment_source)
+      allow_any_instance_of(Spree::CreditCard).to receive(:reusable?).and_return(true)
+
+      expect { subject.add_to_wallet }.to not_change {
```

Comment:
I think you can just use `.to_not change` here, without defining the custom matcher on line 5.